### PR TITLE
Bug 1772694: default template for metering report for editor should use v1

### PR DESCRIPTION
**Pull request by dtaylor113**

### CODE REVIEW

File: frontend/public/models/yaml-templates.ts [hunk 1]
```diff
@@ -194,7 +194,7 @@ spec:
   .setIn(
     [referenceForModel(k8sModels.ChargebackReportModel), 'default'],
     `
-apiVersion: metering.openshift.io/v1alpha1
+apiVersion: metering.openshift.io/v1
```
Comment (spadgett) [inline frontend/public/models/yaml-templates.ts]: `ChargebkacReportModel` still refers to `v1alpha1`. We can't just change the template YAML without changing the model and switching over entirely.

Are there API changes from `v1alpha1` to `v1`?

Comment (dtaylor113) [reply frontend/public/models/yaml-templates.ts]: I also see `v1alpha`:

https://github.com/openshift/console/blob/master/frontend/public/models/index.ts#L47
..and..
https://github.com/openshift/console/blob/master/frontend/public/components/chargeback.tsx#L30

..but I'm not 100% sure what there version number changes impact

Comment (dtaylor113) [reply frontend/public/models/yaml-templates.ts]: I changed `v1alpha` to `v1` in the areas I mentioned above ^^, from a url perspective:
http://0.0.0.0:9000/k8s/ns/openshift-metering/metering.openshift.io~v1~Report
...works, but 
http://0.0.0.0:9000/k8s/ns/openshift-metering/metering.openshift.io~v1~ReportQuery
...results in a `404: Not Found`

http://0.0.0.0:9000/k8s/ns/openshift-metering/metering.openshift.io~v1alpha1~ReportQuery
...continues to work, but have to manually enter `v1alpha1` in the url

Comment (dtaylor113) [reply frontend/public/models/yaml-templates.ts]: Also, when did this change happen?  Does this need to be backported to release-4.2?

Comment (dtaylor113) [reply frontend/public/models/yaml-templates.ts]: Backported changes to a 4.2 cluster and it worked there as well. Will do a followup PR to merge into release-4.2 branch.

File: frontend/public/components/chargeback.tsx [hunk 2]
```diff
@@ -27,8 +27,7 @@ import {
 
 export const ReportReference: GroupVersionKind = referenceForModel(ChargebackReportModel);
 export const ScheduledReportReference: GroupVersionKind = 'metering.openshift.io~ScheduledReport';
-export const ReportGenerationQueryReference: GroupVersionKind =
-  'metering.openshift.io~v1alpha1~ReportQuery';
+export const ReportGenerationQueryReference: GroupVersionKind = referenceForModel(ReportQueryModel);
```
Comment (spadgett) [inline frontend/public/components/chargeback.tsx]: 👍

**Pull request merged**